com.unity.cinemachine/Runtime/Components/CinemachinePanTilt.cs

```diff
@@ -39,6 +39,26 @@ public enum ReferenceFrames
         /// <summary>Defines the reference frame against which pan and tilt rotations are made.</summary>
         public ReferenceFrames ReferenceFrame = ReferenceFrames.ParentObject;
 
+        /// <summary>
+        /// Defines the recentering target: Recentering goes here
+        /// </summary>
+        public enum RecenterTargetModes
```

Comment:
Have you checked with RM if this can go in 3.1? Let me know if you need a new bump configured.